core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -383,8 +383,7 @@ class GroupMetadataManager(brokerId: Int,
                              filteredOffsetMetadata: Map[TopicIdPartition, OffsetAndMetadata],
                              responseCallback: immutable.Map[TopicIdPartition, Errors] => Unit,
                              producerId: Long,
```

Comment:
nit: These parameters are not aligned with the 1st line. 
```
private def createPutCacheCallback(isTxnOffsetCommit: Boolean,
                                   group: GroupMetadata,
                                   ...
```

Reply:
Updated it. Thanks.